Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -1065,7 +1077,12 @@ namespace O3DE::ProjectManager
                 registrationResult = !pythonRegistrationResult.cast<bool>();
             });
 
-        return result && registrationResult;
+        if (!result || !registrationResult)
+        {
+            return AZ::Failure<AZStd::string>(AZStd::move(RedirectOutput::lastPythonError));
```

Comment:
if stored in an QStringList you can join them on a newline character and return that, or if you really just want the last error you can return the last one in the list.